source/SpinalHDL/Simulation/index.rst

```diff
@@ -89,6 +89,8 @@ How SpinalHDL simulates the hardware with Synopsys VCS backend
 
 * Synopsys VCS is a **commercial** simulation tool. It is close source and not free. You have to own the licenses to **legally** use it.
 
+Before using VCS as the simulation backend, make sure that you have checked your system environment as :ref:`VCS environment <_vcs_env>`.
```

Comment:
This give me a warning about undefined label: _vcs_env.
Need to check manually if this is a real concern.

Reply:
Maybe you're right. I'm not so familiar with the syntax of rst. How can I create a reference in rst? I have added a label `_vcs_env` at line 5 in `source/SpinalHDL/Simulation/install/VCS.rst`.

Reply:
Same warning happens in Github check flow...

Reply:
I think is just the sputius underscore.
```suggestion
Before using VCS as the simulation backend, make sure that you have checked your system environment as :ref:`VCS environment<vcs_env>`.
```